draft-ietf-scitt-architecture.md

```diff
@@ -952,6 +958,9 @@ Preventing a Transparency Service from registering Signed Statements that do not
 In contrast, Transparency Services can be held accountable and blamed by an Auditor that replays the Sequence of Signed Statements captured in their Verifiable Data Structure to confirm that a contested Receipt is valid and was correctly registered.
 
 Transparency Services can provide consistency proofs allowing Auditors to check if a set of Receipts were issued from a single Verifiable Data Structure, without replaying individual Signed Statements.
+
+Transparency Services can provide proof allowing Auditors to check that a transparent Statement Sequence is complete and consistent and contains no ommisions.
```

Comment:
```suggestion
Certain Verifiable Data Structures enable a Transparency Service to prove properties of it's Statement Sequence.
For example, proving a specific signed statement is included in the sequence, or that the sequence has only been extended (append only property) since the last time such a proof was created.
```

Reply:
I am fine with, and like, this change